src/admin/importers/BlockImporter.php

```diff
@@ -76,19 +76,27 @@ public function run()
      */
     protected function handleBlockDefinitions(array $definitions)
     {
+        $cwds = [Yii::getAlias('@app'), getcwd()];
+
         $ids = [];
         foreach ($definitions as $blockDefinition) {
+
             $block = Yii::getAlias($blockDefinition, false);
             // if there is no alias, or not found, switch back to original name
             if ($block === false) {
                 $block = $blockDefinition;
             }
-            if (is_file($block)) {
-                $ids[] = $this->saveBlockByPath($block);
-            } elseif (is_dir($block)) {
-                $ids = array_merge($ids, $this->saveBlocksFromFolder($block));
-            } else {
-                $this->addLog("Unable to process block definition '{$block}'");
+
+            foreach ($cwds as $prefix) {
```

Comment:
Is it safe to search the block path twice? When I already in app path the ids will contains duplicates. I think it should break out of this loop as soon as a path has found.

Reply:
Yes i total agree, its not finished. I also want to preserve that the path appears two times in the log. I just tested this in order to fix: https://github.com/luyadev/luya-composer/issues/10

thanks for the review